### PR TITLE
#1: added initial project version

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
 # hamcrest-resultset-matcher
-Machter for SQL resultsets
+Matcher for SQL resultsets
+
+# Overview
+
+This project provides hamcrest matcher that compares java.sql.ResultSet objects.
```
Comment (redcatbear) [inline README.md]:
```suggestion
This project provides hamcrest matcher that compares `java.sql.ResultSet` objects.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
 # hamcrest-resultset-matcher
```
Comment (redcatbear) [inline README.md]: Add newline.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+
+### Run Time Dependencies
+
+Running the Hamcrest ResultSet Matcher requires a Java Runtime version 11 or later.
+
+### Build Time Dependencies
+
+| Dependency                                                                          | Purpose                                                | License                       |
+|-------------------------------------------------------------------------------------|--------------------------------------------------------|-------------------------------|
+| [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                                  | Checking for conditions in code via matchers           | BSD License                   |
```
Comment (redcatbear) [inline README.md]: This is a runtime dependency in this case.

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.exasol</groupId>
+    <artifactId>hamcrest-resultset-matcher</artifactId>
+    <version>${product.version}</version>
```
Comment (redcatbear) [inline pom.xml]:
```suggestion
    <version>0.1.0</version>
```

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    <name>Matcher for SQL Result Sets</name>
+    <description>This project provides hamcrest matcher that compares java.sql.ResultSet objects.</description>
+    <url>https://github.com/exasol/hamcrest-resultset-matcher</url>
+    <properties>
+        <product.version>0.1.0</product.version>
```
Comment (redcatbear) [inline pom.xml]: Remove this line.

File: README.md (hunk 1 above)
Comment (AnastasiiaSergienko) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (AnastasiiaSergienko) [reply README.md]: done

File: pom.xml (hunk 5 above)
Comment (AnastasiiaSergienko) [reply pom.xml]: done

**Pull request merged**